app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -243,7 +246,23 @@ class HomeFragment : Fragment() {
         if (requireContext().settings().pocketRecommendations) {
             lifecycleScope.async(IO) {
```

Comment:
nit: Unrelated to this PR, but wonder why we are using `async()` here since we do not capture any result. This could be `launch`?

Reply:
Definitely!